Java/src/test/java/com/Bug/Bug1

```diff
@@ -0,0 +1,14 @@
+Title: Email Address Field in contact us section accepts invalid email address.
+
+Description: When i enter any invalid email address for example abc@abc.com, input field accepts it.
+
+Steps to reproduce: Go to "Contact Us" in the navigation bar, 

```

Comment:
Make your steps more verbose.
They should be in a numbered list (1,2,3,etc).
Include an expected result for each step.
Since this is a web app you can include the browser version and type.
In practice we can include traceability to the feature or the testcase.